roles/openshift_master/tasks/systemd_units.yml

```diff
@@ -0,0 +1,61 @@
+# This file is included both in the openshift_master role and in the upgrade
+# playbooks.  For that reason the ha_svc variables are use set_fact instead of
+# the vars directory on the role.
+
+- debug: var=openshift_version
+
+- name: Init HA Service Info
+  set_fact:
+    ha_svc_svc_dir: "/usr/lib/systemd/system"
```

Comment:
I'd like to ship them in the RPMs, I don't see any technical blockers to doing so. I just haven't made time to do the work.